src/redux/migrations.ts

```diff
@@ -1861,4 +1861,23 @@ export const migrations = {
     ...state,
     account: _.omit(state.account, 'pictureUri'),
   }),
+  228: (state: any) => {
+    const transactionsByNetworkId: any = {}
+    for (const networkId of Object.keys(state.transactions.transactionsByNetworkId)) {
+      const transactions = []
+      for (const tx of state.transactions.transactionsByNetworkId[networkId]) {
+        tx.providerId && tx.providerId === 'aave-v3'
```

Comment:
I guess only earn transactions can have providerId, should we check those?

Reply:
I was using the check for `tx.providerId` as a proxy for checking that it was an earn transaction, but I could add an explicit check for the type of transaciton.